gtk/src/toga_gtk/libs/gtk.py

```diff
@@ -23,6 +23,9 @@
     Gtk.get_micro_version(),
 )
 
+if GTK_VERSION >= (4, 0, 0):  # pragma: no-cover-if-gtk3
+    from gi._gi import hook_up_vfunc_implementation  # noqa: E402, F401
+
 if GTK_VERSION < (4, 0, 0):  # pragma: no-cover-if-gtk4
```

Comment:
Again - this is the same comparison, in directly seqeuential blocks. You can combine them.